internal/config/cgmgr/systemd.go

```diff
@@ -46,6 +46,24 @@ func (*SystemdManager) ContainerCgroupPath(sbParent, containerID string) string
 	return parent + ":" + crioPrefix + ":" + containerID
 }
 
+// ContainerCgroupAbsolutePath takes arguments sandbox parent cgroup and container ID and
+// returns the cgroup path on disk for that containerID. If parentCgroup is empty, it
+// uses the default parent system.slice
+func (r *SystemdManager) ContainerCgroupAbsolutePath(sbParent, containerID string) (string, error) {
```

Comment:
hm, why `r` here?

Reply:
the linter wanted it
it used to be an m, but the linter was like "why is this an m, everything else is an r"

Reply:
I think we should change all cases to `m`, `r` has less meaning here IMO

Reply:
can do lol